minecraft4k.txt

```diff
@@ -0,0 +1,14 @@
+[Section]
+Name = Minecraft 4K
+Version = v2
+License = Apache
+Description = Minecraft 4k was an edition of Minecraft developed for the 2010 Java 4K contest.
+Category = 4
+URLSite = https://maverk-corporation.itch.io/minecraft4k
+URLDownload = https://github.com/fiatcoupe16vturbo/Minecraft-4K-Mirror/raw/refs/heads/main/Minecraft4K.jar
+SHA1 = 49f7ae310b48dd41266a2e01c91cfb5325dda0ee
+SizeBytes = 16384
```

Comment:
I do doubt that Minecraft would be only 16kByte in size.

Reply:
If I download that file, then the size is 9.852 Bytes for me:
![image](https://github.com/user-attachments/assets/b97d2093-c667-4825-9d6d-e5d63b31cadf)
So please fix that value.